src/main/java/com/github/theholywaffle/teamspeak3/TS3ApiAsync.java

```diff
@@ -3038,6 +3093,27 @@ public void handle() {
 	}
 
 	/**
+	 * Moves a list of clients into the specified channel using the specified password.
+	 *
+	 * @param clients
+	 * 		the list of clients to move.
+	 * @param channel
+	 * 		the channel to move the client into
+	 * @param channelPassword
+	 * 		the password of the channel
+	 * @return whether the command succeeded or not
+	 *
+	 * @querycommands 1
+	 */
+	public CommandFuture<Boolean> moveClient(List<Client> clients, ChannelBase channel, String channelPassword) {
+		List<Integer> clientIds  = null;
```

Comment:
NullPointerException as in TS3Api; plus superfluous space (`  = ` instead of ` = `)